examples/Raylib/CMakeLists.txt

```diff
@@ -3,14 +3,13 @@ if (POLICY CMP0074)
     cmake_policy(SET CMP0074 NEW)
 endif()
 
-project(LDtkLoader-Raylib)
-
-set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_PATH})
+project(LDtkLoader-raylib)
 
 find_package(raylib REQUIRED)
+
 add_executable(LDtkRaylib main.cpp)
-target_include_directories(LDtkRaylib PRIVATE ../../include ${RAYLIB_INCLUDE_DIRS})
-target_link_libraries(LDtkRaylib PRIVATE ${RAYLIB_LIBRARIES} LDtkLoader)
+target_include_directories(LDtkRaylib PRIVATE ../../include ${raylib_INCLUDE_DIRS})
+target_link_libraries(LDtkRaylib PRIVATE ${raylib_LIBRARIES} LDtkLoader)
```

Comment:
After this change, this no longer compiles on MSVC, after reverting to the old commit works fine. I think it's due to vcpkg setting those capitalized https://github.com/microsoft/vcpkg/blob/030cfaa24de9ea1bbf0a4d9c615ce7312ba77af1/ports/raylib/vcpkg-cmake-wrapper.cmake#L10
Maybe something like this could solve the issue.
```cmake
if (MSVC)
    if(NOT DEFINED ${raylib_INCLUDE_DIRS})
        set(raylib_INCLUDE_DIRS ${RAYLIB_INCLUDE_DIRS})
    endif()

    if(NOT DEFINED ${raylib_LIBRARIES})
        set(raylib_LIBRARIES ${RAYLIB_LIBRARIES})
    endif()
endif()
```

Reply:
Ah, so that's the reason ! Looks good, you can add this change.
Then I'll merge later today !